src/common/apiData/genres/genresSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const genresSlice = createSlice({
+  name: "genres",
+  initialState: {
+    status: "initial",
```

Comment:
zastanawiam się czy status jest tu w ogóle potrzebny, skoro mamy isGenres?

Reply:
Myślałem o statusie w celu obsłużenia ładowania. Może dodać jakiś mały odpowiednik loadera w miejscu gdzie wyświetlają się gatunki. Jeśli nie chcemy tego zrobić to rzeczywiście, status można usunąć.